test/constraints/constraint_unit.jl

```diff
@@ -235,6 +270,41 @@ function test_constraint_unit_state_transition()
     end
 end
 
+function test_units_out_of_service_transition()
+    @testset "constraint_unit_state_transition" begin
```

Comment:
Is it ok to reuse a @testset name? I am not sure they both get executed in that case - could you check?

Reply:
Fix this just to be sure in [latest commit](https://github.com/spine-tools/SpineOpt.jl/commit/21097e41186dd36e83125e591bcfd34989910eeb)